Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -261,6 +261,11 @@ void VideoConfig::VerifyValidity()
       iStereoMode = 0;
     }
   }
+  if (!backend_info.bSupportsBBox && g_ActiveConfig.bBBoxEnable)
+  {
+    PanicAlertT("Your graphics card does not support bounding box emulation this game requires. "
+                "You will experience bugs and crashes. This is not a Dolphin bug.");
```

Comment:
I agree with @JosJuice that the wording should be changed. The current version is a bit awkward.